<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeFormatEditWidget</class>
 <widget class="QWidget" name="TimeFormatEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="valueTypeGroupBox">
     <property name="title">
      <string>Value type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="showElapsedRadioButton">
        <property name="text">
         <string>Show elapsed time</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="showActualRadioButton">
        <property name="text">
         <string>Show actual time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="elapsedTimeFormatgroupBox">
     <property name="title">
      <string>Elapsed time format</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="elapsedSSRadioButton">
        <property name="text">
         <string>SS sec</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="elapsedMMSSRadioButton">
        <property name="text">
         <string>MM:SS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="elapsedHHMMSSRadioButton">
        <property name="text">
         <string>HH:MM:SS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="elapsedDDHHMMSSRadioButton">
        <property name="text">
         <string>DD:HH:MM:SS</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="actualTimeFormatGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Actual time format</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="actualYYYYMMDDHHHHMMSSRadioButton">
        <property name="text">
         <string>yyyy/mm/dd HH:MM:SS</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="actualYYYYMMDDHHHHMMRadioButton">
        <property name="text">
         <string>yyyy/mm/dd HH:MM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="actualMMDDHHMMSSRadioButton">
        <property name="text">
         <string>mm/dd HH:MM:SS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="actualMMDDHHMMRadioButton">
        <property name="text">
         <string>mm/dd HH:MM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="actualHHMMSSRadioButton">
        <property name="text">
         <string>HH:MM:SS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="actualHHMMRadioButton">
        <property name="text">
         <string>HH:MM</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>showElapsedRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>elapsedTimeFormatgroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showActualRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>actualTimeFormatGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
